Marlin/src/module/temperature.cpp

```diff
@@ -583,7 +591,7 @@ volatile bool Temperature::raw_temps_ready = false;
         updateTemperaturesFromRawValues();
 
         // Get the current temperature and constrain it
-        current_temp = GHV(temp_chamber.celsius, temp_bed.celsius, temp_hotend[heater_id].celsius);
+        current_temp = (float)GHV(temp_chamber.celsius, temp_bed.celsius, temp_hotend[heater_id].celsius);
```

Comment:
Compiler warning?

Reply:
The right part is a celsius_t  but the current_temp is a float (I've kept it as float because I did not spend time studying the required precision for the later computation). When I'll have finished the change, either it'll be celsius_t (if a huge precision is not required) or it'll stay as float. 

Don't spend too much time on those, I'm currently merging all 4 similar functions into one here so it'll save flash space.